Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -116,67 +128,422 @@ sample_triangles(const TriangleRange& triangles, double distance, OutputIterator
+ *
+ * @param tm1 the triangle mesh that will be sampled
+ * @param tm2 the triangle mesh to compute the distance to
+ * @param precision the number of points per squared area unit for the random sampling
+ * @param np1 optional sequence of \ref namedparameters for `tm1` among the ones listed below
+ * @param np2 optional sequence of \ref namedparameters for `tm2` among the ones listed below
+ *
+ * \cgalNamedParamsBegin
+ *    \cgalParamBegin{vertex_point_map} the property map with the points associated to the vertices of `pmesh` \cgalParamEnd
+ *    \cgalParamBegin{geom_traits} an instance of a geometric traits class, model of `Kernel`\cgalParamEnd
+ * \cgalNamedParamsEnd
+ */
+template< class Concurrency_tag,
+          class TriangleMesh,
+          class NamedParameters1,
+          class NamedParameters2>
+double approximated_Hausdorff_distance( const TriangleMesh& tm1,
+                                        const TriangleMesh& tm2,
+                                        double precision,
+                                        const NamedParameters1& np1,
+                                        const NamedParameters2& np2)
+{
+  typedef typename GetGeomTraits<TriangleMesh,
+                                 NamedParameters1>::type Geom_traits;
+
+  return approximated_Hausdorff_distance<Concurrency_tag, Geom_traits>(
+    tm1, tm2,
+              precision,
+              choose_param(get_param(np1, vertex_point),
+                           get_const_property_map(vertex_point, tm1)),
+
+              choose_param(get_param(np2, vertex_point),
+                           get_const_property_map(vertex_point, tm2))
+
+  );
+}
+
+/**
+ * \ingroup PMP_distance_grp
+ * computes the approximated symmetric Hausdorff distance between `tm1` and `tm2`.
+ * It returns the maximum of `approximated_Hausdorff_distance(tm1,tm2)` and
+ * `approximated_Hausdorff_distance(tm1,tm2)`.
+ *
+ * \copydetails CGAL::Polygon_mesh_processing::approximated_Hausdorff_distance()
+ */
+template< class Concurrency_tag,
+          class TriangleMesh,
+          class NamedParameters1,
+          class NamedParameters2>
 double approximated_symmetric_Hausdorff_distance(
-  const TriangleRange1& triangles_1,
-  const TriangleRange2& triangles_2,
-  double targeted_precision
-)
+  const TriangleMesh& tm1,
+  const TriangleMesh& tm2,
+  double precision,
+  const NamedParameters1& np1,
+  const NamedParameters2& np2)
 {
   return (std::max)(
-    approximated_Hausdorff_distance<Kernel>(triangles_1, triangles_2, targeted_precision),
-    approximated_Hausdorff_distance<Kernel>(triangles_2, triangles_1, targeted_precision)
+    approximated_Hausdorff_distance<Concurrency_tag>(tm1,tm2,precision,np1,np2),
+    approximated_Hausdorff_distance<Concurrency_tag>(tm2,tm1,precision,np2,np1)
   );
 }
 
+/**
+ * \ingroup PMP_distance_grp
+ * computes the approximated Hausdorff distance between `points` and `tm`.
```

Comment:
Yes it is, there is a precision to be given somewhere (at least there was in my code).